lib/features/support/app-automate.rb

```diff
@@ -0,0 +1,63 @@
+require 'appium_lib'
+require_relative './driver'
+
+class AppAutomateDriver < Driver
+
+  attr_reader :device_type
+
+  APP_UPLOAD_URI = "https://api-cloud.browserstack.com/app-automate/upload"
+
+  def devices
+    Devices::DEVICE_HASH
+  end
+
+  def start_driver(target_device, app_location)
```

Comment:
What is this method called by?

Reply:
This is called by the environment of the feature set being tested.  It could be called from the driver initialisation, but I'd like the drivers to be reasonably "dumb" and controllable.